ignite/contrib/handlers/lr_finder.py

```diff
@@ -229,11 +232,30 @@ def plot(
         if skip_end < 0:
             raise ValueError("skip_end cannot be negative")
 
-        # Get the data to plot from the history dictionary. Also, handle skip_end=0
-        # properly so the behaviour is the expected
-
+        # Get the data to plot from the history dictionary.
         lrs = self._history["lr"]
         losses = self._history["loss"]
+
+        # Check to show the suggested learning rate
+        if display_suggestion:
+            sug_lr = self.lr_suggestion()
+            idx = self._history["lr"].index(sug_lr)
+
+            if skip_start >= idx:
+                warnings.warn(
+                    "skip_start is large, that would make the plot less clear,"
+                    " and it may affects displaying the suggestion",
```

Comment:
```suggestion
                    "skip_start is larger then the suggested LR found"
                    " and it will not be visible on the plot. Please, make the value smaller.",
```